### DISCUSSION TITLE
Long touch press on marker with tooltip

### DISCUSSION THREAD

**Discussion by timokoessler** [Leaflet/Leaflet#9080] [Q&A]
Hello,

When I long tap on a marker on a mobile device (Chrome on Android in my case), the tooltip displays correctly, but the browser opens a popup to share or save the image. Is there any way to prevent this behavior of the browser so that the tooltip is still displayed?

Thank you,
Timo

**Comment by Falke-Design** [comment]
There is already a open issue but we have no solution for this so far. #6865
